web/themes/custom/votegov/templates/node/node--state-territory.html.twig

```diff
@@ -203,7 +203,9 @@
     {{ mail_registration.text }}
 
     {% if state_mail_pdf_link and mail_registration.link_text %}
-      <p>{{ link(mail_registration.link_text, state_mail_pdf_link, create_attribute().setAttribute('data-state-ext', 'state-mail-registration')) }}</p>
+      <p>{{ link(mail_registration.link_text, state_mail_pdf_link, create_attribute()
+            .setAttribute('data-state-ext', 'state-mail-registration')
+            .addClass('usa-link--external')) }}</p>
```

Comment:
The external links script is not targeting the anchor within the link() class so I added the class here manually. In the code, some links are using this class, while others are using the anchor tag directly. Is there a reason to use one over the other?